src/config/navigationConfig.js

```diff
@@ -15,6 +15,8 @@ export const categoryPageFilterAttribute = 'hierarchicalCategories.lvl0';
 // The name is for display purposes for each navigation title
 // The filter is the value to filter on to show results for the linked category page
 // The url should be the lower case URL friendly version of the filter value
+// Please use filterComplex type and moreComplexFilter attribute to use the filter with complexity
```

Comment:
You can say instead something like "if you need to define your own filter attributes and values, use rawFilters, instead use "filter". Remember that filter will use the categoryPageAttribute."

Reply:
Thank you @matthew1809 for the correction 🙏

Reply:
Do you think I have to change key name in the object? 
![Screenshot 2022-08-24 at 12 29 18](https://user-images.githubusercontent.com/22235112/186396342-14415ebc-a298-48c4-b18e-ccfd0a82ac17.png)